hsctl

```diff
@@ -60,8 +60,9 @@ restart_hs() {
     echo "*** ${1^^} ***"
     stop_nginx
     preflight_hs
-    docker stop hydroshare_hydroshare_1
-    docker start hydroshare_hydroshare_1
+    docker-compose stop hydroshare
+    docker-compose build hydroshare
```

Comment:
@mjstealey Do we need to run "docker-compose build" for restart_hs()? Is 'docker-compose up' followed by 'docker-compose stop' sufficient?